python-calculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
Hi @lrbarn,

I think there a typo here in the function definition.
```suggestion
def divide(x, y):
```

I think this should work here

Reply:
good spot, your fix looks correct to me